### PR TITLE
Add possibility to delete Default Reviewers

### PR DESCRIPTION
**Pull request by j0nathan33**
My last PR for next few week :)

### CODE REVIEW

**Comment by cdancy** [comment]
> My last PR for next few week :)

Don't ever feel bad about sending in code/PR's. The more the better ;)

Also ... It's going to be a few days until I can get to these. I'm fixing integration tests with this project so that they can all just run instead of the silly behavior we have now. Once that is done it will making testing changes much easier and reduce the PR turnaround time. Stay tuned.

**Comment by j0nathan33** [comment]
It's not that. I am going on vacation

File: src/main/java/com/cdancy/bitbucket/rest/domain/branch/Matcher.java [hunk 1]
```diff
@@ -25,8 +25,10 @@
 
     public enum MatcherId {
         RELEASE("RELEASE", "Release", "MODEL_CATEGORY", "Branching model category"),
```
Comment (cdancy) [inline src/main/java/com/cdancy/bitbucket/rest/domain/branch/Matcher.java]: Think we should also switch `RELEASE` and `Release` around as well?

Comment (j0nathan33) [inline src/main/java/com/cdancy/bitbucket/rest/domain/branch/Matcher.java]: For that, it's not possible to switch it because bitbucket have valid for that

**Comment by cdancy** [comment]
Closing as this has been merged as part of #92

**Comment by cdancy** [comment]
Thanks again @j0nathan33 !

**Pull request closed without merging**